certbot/docs/cli-help.txt

```diff
@@ -99,9 +99,9 @@ optional arguments:
                         before submitting to CA (default: False)
   --preferred-chain PREFERRED_CHAIN
                         If the CA offers multiple certificate chains, prefer
-                        the chain with an issuer matching this Subject Common
-                        Name. If no match, the default offered chain will be
-                        used. (default: None)
+                        the chain whose topmost certificate was issued from
+                        this Subject Common Name. If no match, the default
+                        offered chain will be used. (default: None)
```

Comment:
This file is automatically generated during the release process, the place to change it in the source is [here](https://github.com/certbot/certbot/blob/7a02deeeba4e823c9b947a311fd85a5497a9dd30/certbot/certbot/interfaces.py#L264-L268).

It's a little sad that there's no friendlier way to phrase this help text, but hopefully it doesn't matter too much since users will likely be copy-pasting instructions.

Reply:
Ooh thanks for the pointer; fixed! I tried but failed to come up with better phrasing.